config/metrics_definitions/config.yaml

```diff
@@ -0,0 +1,21 @@
+## This is the main configuration file for flowlogs-pipeline. It holds
```

Comment:
can we remove this file? I don't see why it would be needed, this is not where the FLP pipeline config should be

Reply:
removed. confgenerator updated to accomodate missing config.yaml in case of GenerateStages.